Toggl.Daneel/Extensions/Reactive/UIViewExtensions.cs

```diff
@@ -66,5 +67,13 @@ public static Action<float> AnimatedAlpha(this IReactive<UIView> reactive)
                 );
             };
 
+        public static IDisposable BindAction(this IReactive<UIView> reactive, UIAction action)
+        {
+            return Observable.Using(
+                    () => { return action.Enabled.Subscribe(e => { reactive.Base.UserInteractionEnabled = e; }); },
```

Comment:
I know you like `return`s but I think here they are redundant:
```suggestion
                    () => action.Enabled.Subscribe(e => { reactive.Base.UserInteractionEnabled = e; }),
```